EMBEDDED_PROTOCOL_CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 4.0.0
```

Comment:
According to the embedded protocol versioning policy, this isn't a breaking change because the wire protocol is backwards-compatible from the host's perspective. It should be 3.3.0.